front/src/common/api/generatedEditoastApi.ts

```diff
@@ -2449,46 +2433,74 @@ export type PathItemLocation =
       /** The [UIC](https://en.wikipedia.org/wiki/List_of_UIC_country_codes) code of an operational point */
       uic: number;
     };
-export type PathfindingResult =
-  | (PathfindingResultSuccess & {
-      status: 'success';
-    })
+export type PathfindingInputError =
+  | {
+      error_type: 'invalid_path_items';
+      items: {
+        index: number;
+        path_item: PathItemLocation;
+      }[];
+    }
+  | {
+      error_type: 'not_enough_path_items';
+    }
+  | {
+      error_type: 'rolling_stock_not_found';
+      rolling_stock_name: string;
+    };
+export type OffsetRange = {
+  end: number;
+  start: number;
+};
+export type IncompatibleOffsetRangeWithValue = {
+  range: OffsetRange;
+  value: string;
+};
```

Comment:
Is there any reason why this type can't extend `IncompatibleOffsetRange` with the `value` property ?

Reply:
We can not do that in editoast.

Reply:
When you do that, it's the same thing as I'm thinking
![image](https://github.com/user-attachments/assets/0c235377-542a-4e77-b820-8538e256bdc4)

This could be 
```suggestion
export type IncompatibleOffsetRangeWithValue = IncompatibleOffsetRange & {
  value: string;
};
```

But if it's complicated, it's not very important :slightly_smiling_face:

Reply:
This is a generated file from Rust code. There is no such thing as inheritance or type extensions in Rust.